src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -21,27 +21,16 @@ class Section
     /**
      * @var StopwatchEvent[]
      */
-    private $events = [];
+    private array $events = [];
 
-    /**
-     * @var float|null
-     */
-    private $origin;
-
-    /**
-     * @var bool
-     */
-    private $morePrecision;
-
-    /**
-     * @var string
-     */
-    private $id;
+    private ?float $origin;
+    private bool $morePrecision;
+    private ?string $id = null;
```

Comment:
should we default it to `$this->id = '';` inside the constructor?
nullable seems weird :s

Reply:
I didn't want to change the behavior here.

Reply:
understood :)